### ISSUE TITLE
Ability to clone a collection schema (without documents) and it's synonyms and overrides

### ISSUE THREAD

**Issue by jasonbosco** [typesense/typesense#826]

**Comment by jasonbosco** [comment]
This is now supported in `0.24.0.rcn42`:

```
curl -k "http://localhost:8108/collections?src_name=existing_collection_name" -X POST -H "Content-Type: application/json" \
      -H "X-TYPESENSE-API-KEY: ${TYPESENSE_API_KEY}" -d '{
        "name": "new_collection_name"
      }'
```

**Comment by jasonbosco** [comment]
This is now available in v0.24: https://typesense.org/docs/0.24.0/api/collections.html#cloning-a-collection-schema
